modules/custom/cu_newsletter/cu_newsletter.module

```diff
@@ -494,6 +494,24 @@ function cu_newsletter_design_options() {
  *  Implements hook_form_FORM_ID_alter.
  */
 function cu_newsletter_form_newsletter_node_form_alter(&$form, &$form_state, $form_id) {
+  // Check to see if there are any newsletter type terms
+  // Display warning if there is no terms
+  if (count($form['field_newsletter_type'][LANGUAGE_NONE]['#options']) < 2) {
+    $link = l('create a newsletter type', 'admin/structure/taxonomy/newsletter/add');
```

Comment:
It would be nice to add "?destination=node/add/newsletter" to the link so you can get back to where you started.